mltsp/datasets/tests/test_datasets.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import print_function
+import os
+
+import numpy as np
+import numpy.testing as npt
+from sklearn.externals import joblib
+
+from mltsp import datasets as ds
+
+
+def test_fetch_andrzejak(allow_download=False):
```

Comment:
So that functionality never gets tested? Then we at least need a small mock
dataset to test downloading with.

On Fri, Dec 18, 2015 at 11:55 AM, Brett Naul <notifications@github.com>
wrote:

> In mltsp/datasets/tests/test_datasets.py
> <https://github.com/mltsp/mltsp/pull/124#discussion_r48063200>:
>
> > @@ -0,0 +1,26 @@
> > +from __future__ import print_function
> > +import os
> > +
> > +import numpy as np
> > +import numpy.testing as npt
> > +from sklearn.externals import joblib
> > +
> > +from mltsp import datasets as ds
> > +
> > +
> > +def test_fetch_andrzejak(allow_download=False):
>
> I don't think so? It takes forever...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mltsp/mltsp/pull/124/files#r48063200>.
>